docs/source/indepth_tutorial/mpi-evaluator.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>if you use conda, in my experience installing mpi4py adds mpi as well, so at least on Mac with conda there is no need to get brew or so involved.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Nice addition, added a note.